akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingEvents.scala

```diff
@@ -53,7 +58,8 @@ private[akka] object ReplayingEvents {
       receivedPoisonPill: Boolean,
       recoveryStartTime: Long,
       version: VersionVector,
-      seenSeqNrPerReplica: Map[ReplicaId, Long])
+      seenSeqNrPerReplica: Map[ReplicaId, Long],
+      eventsRecovered: Long)
```

Comment:
rename to `eventsReplayed`

we try to use "replay" for the events, and "recover" for the full process of snapshot + replay